frontend/src/utils/format_column_data.ts

```
@@ -63,35 +57,121 @@ export class GraphData {
+        ) {
+          this.canvas.clearRect(x, y, 10, 10);
+          this.canvas.fillStyle = "grey";
+          this.canvas.beginPath();
+
+          this.canvas.arc(x, y, 10, 0, Math.PI * 2);
+          this.canvas.fill();
+
+          // Make the child connections all light up
+          for (const [l_x, l_y] of this.drawable_shapes[i][3]) {
+            const cx = x,
+              cy = l_y;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)